tests/kafkatest/tests/connect/connect_distributed_test.py

```diff
@@ -155,7 +155,43 @@ def test_restart_failed_connector(self):
         wait_until(lambda: self.connector_is_running(self.sink), timeout_sec=10,
                    err_msg="Failed to see connector transition to the RUNNING state")
 
-    
+    @matrix(delete_before_reconfig=[False, True])
+    def test_bad_connector_class(self, delete_before_reconfig):
```

Comment:
This is probably reasonable as a system test, but would it make sense to have a unit test as well?